app/src/sharedTest/java/org/oppia/app/story/StoryFragmentTest.kt

```diff
@@ -46,14 +47,14 @@ class StoryFragmentTest {
 
   @Test
   fun testStoryFragment_clickOnToolbarNavigationButton_closeActivity() {
-    launch<StoryFragmentTestActivity>(createTestActivityIntent(TEST_STORY_ID_1)).use {
+    launch<StoryFragmentTestActivity>(createTestActivityIntent(0, TEST_TOPIC_ID_0, TEST_STORY_ID_1)).use {
```

Comment:
Declare constant for internal profile Id here and wherever required.

Reply:
Updated. Thanks.